### PR TITLE
syncthing 1.7.1

### PR DESCRIPTION
**Pull request by Kriechi**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Kriechi** [comment]
I think https://github.com/syncthing/syncthing/pull/6689/files broken the homebrew build step for syncthing.
@calmh any idea what the correct invocation of the `build.sh` script should be now?

**Comment by timfallmk** [comment]
Looks like they removed a bunch of "cruft" that was considered unused. My recommendation would be to skip the build script and instead invoked the call directly. Something like:
```shell
go run build.go --no-upgrade --tar
```
Would be my guess

File: Formula/syncthing.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def install
     src = buildpath/"src/github.com/syncthing/syncthing"
     src.install buildpath.children
     src.cd do
-      system "./build.sh", "noupgrade"
+      system "./build.sh"
```
Comment (timfallmk) [inline Formula/syncthing.rb:25]:
```suggestion
      system "go", "run", "build.go", "--no-upgrade", "tar"
```

**Comment by calmh** [comment]
Sorry for the breakage, that script was always a shortcut for humans and most of the removed things indeed didn't make sense any more. You probably want `go run build.go -no-upgrade tar`, more info at https://docs.syncthing.net/dev/building.html

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @timfallmk, @calmh!!

**Pull request closed without merging**